src/pages/home/report/ReportActionListFrozenScrollContext.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const ReportActionListFrozenScrollContext = React.createContext({
+    setShouldFreezeScroll: () => {},
+});
```

Comment:
Let's keep value and setter in one place.
```suggestion
const ReportActionListFrozenScrollContext = React.createContext({
    isScrollFrozen: false,
    setShouldFreezeScroll: () => {},
});
```